packages/artifactor/package.json

```diff
@@ -7,8 +7,9 @@
   "repository": "https://github.com/trufflesuite/truffle/tree/master/packages/artifactor",
   "version": "4.0.60",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
-    "build": "tsc",
+    "build": "tsc --declaration",
```

Comment:
Shouldn't this go in the tsconfig?

Reply:
Oh, huh, didn't know you could do that.  Will change that.

Reply:
You'll find it's already there in a few places!